### ISSUE TITLE
intermittent bug with `session`

### ISSUE THREAD

**Issue by mocxd** [GovLab/noi2#354]
sometimes does not save whether you have hit /activity in session properly.. could use some help from others to figure out steps to recreate, and possible help @naffis if u can.  i can't recreate ;(

**Comment by naffis** [comment]
I wasn't able to recreate and didn't see anything obviously causing this int he code. Do you know the steps I can use to recreate the issue described?

**Comment by mocxd** [comment]
yeah same. i cant figure it out.  is there a way to easily monitor the contents of the session that we could just have ppl run who are experiencing ?

**Comment by naffis** [comment]
We could probably have them run something like charles proxy and record their session. That would give us some information from the browser side at least.

**Comment by mocxd** [comment]
has anybody experienced this issue again @chrswng ? do we know anything more about steps to reproduce

**Comment by mocxd** [comment]
possibly triggered by using 'back' button onto activity route, need to investigate
